helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -793,13 +793,14 @@ void createClient() throws Exception {
 
   @Override
   public void connect() throws Exception {
-    LOG.info("ClusterManager.connect()");
```

Comment:
It's still useful to have this log if we return if we are already connected.

Reply:
We already log a warn message on 797 if we are already connected before we return.

Reply:
Ahh completely missed it.